web_widget_number_ux_choice/static/src/xml/number_ux_choice.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
+@author: Quentin DUPONT <quentin.dupont@grap.coop>
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<templates>
+    <t t-name="NumberUxChoice">
```

Comment:
It's a common practice to prefix it with the module name.

```suggestion
    <t t-name="web_widget_number_ux_choice.field">
```

If the module name changes, adapt it here, of course.

Reply:
done !